src/install.ts

```diff
@@ -160,33 +158,20 @@ export async function fetchVersion (edgeVersion: string) {
   throw new Error(`Couldn't detect version for ${edgeVersion}`)
 }
 
-function downloadZip(body: NodeJS.ReadableStream, cacheDir: string) {
-  const stream = Readable.from(body).pipe(unzipper.Parse())
-  const promiseChain: Promise<string | void>[] = [
-    new Promise((resolve, reject) => {
-      stream.on('close', () => resolve())
-      stream.on('error', () => reject())
-    })
-  ]
-
-  stream.on('entry', async (entry: Entry) => {
-    const unzippedFilePath = path.join(cacheDir, entry.path)
-    if (entry.type === 'Directory') {
+async function downloadZip(res: Awaited<ReturnType<typeof fetch>>, cacheDir: string) {
+  const zipBlob = await res.blob()
+  const zip = new ZipReader(new BlobReader(zipBlob))
+  for (const entry of await zip.getEntries()) {
+    const unzippedFilePath = path.join(cacheDir, entry.filename)
+    if (entry.directory) {
       return
```

Comment:
I think this should be `continue` as the zip file contains multiple files and if the driver binary is not the first one, nothing will be written to disk.

Reply:
Ah that's it. Thanks for the hint!